docs/en/rules/Azure.MySQL.GeoRedundantBackup.md

```diff
@@ -0,0 +1,103 @@
+
+**Important** Configuring locally redundant or geo-redundant storage for backup is only allowed during server create. Once the server is provisioned, you cannot change the backup storage redundancy option. In order to move your backup storage from locally redundant storage to geo-redundant storage, creating a new server and migrating the data using dump and restore is the only supported option.
+
+## RECOMMENDATION
+
+Configure geo-redundant backup for Azure Database for MySQL.
+
+## EXAMPLES
+
+### Configure with Azure template
+
+To deploy Azure Database for MySQL servers that pass this rule:
+
+- Set the `properties.storageProfile.geoRedundantBackup` property to the value `'Enabled'`.
+
+For example:
+
+```json
+{
+  "type": "Microsoft.DBforMySQL/servers",
+  "apiVersion": "2017-12-01",
+  "name": "[parameters('serverName')]",
+  "location": "[parameters('location')]",
+  "sku": {
+    "name": "[parameters('skuName')]",
+    "tier": "GeneralPurpose",
+    "capacity": "[parameters('skuCapacity')]",
+    "size": "[format('{0}', parameters('SkuSizeMB'))]",
+    "family": "[parameters('skuFamily')]"
+  },
+  "properties": {
+    "createMode": "Default",
+    "version": "[parameters('mysqlVersion')]",
+    "administratorLogin": "[parameters('administratorLogin')]",
+    "administratorLoginPassword": "[parameters('administratorLoginPassword')]",
+    "storageProfile": {
+      "storageMB": "[parameters('SkuSizeMB')]",
+      "backupRetentionDays": 7,
+      "geoRedundantBackup": "Enabled"
+    }
+  }
+}
+```
+
+### Configure with Bicep
+
+To deploy Azure Database for MySQL servers that pass this rule:
+
+- Set the `properties.storageProfile.geoRedundantBackup` property to the value `'Enabled'`.
+
+For example:
+
+```bicep
+resource mysqlDbServer 'Microsoft.DBforMySQL/servers@2017-12-01' = {
+  name: serverName
+  location: location
+  sku: {
+    name: skuName
+    tier: 'GeneralPurpose'
+    capacity: skuCapacity
+    size: '${SkuSizeMB}'
+    family: skuFamily
+  }
+  properties: {
+    createMode: 'Default'
+    version: mysqlVersion
+    administratorLogin: administratorLogin
+    administratorLoginPassword: administratorLoginPassword
+    storageProfile: {
+      storageMB: SkuSizeMB
+      backupRetentionDays: 7
+      geoRedundantBackup: 'Enabled'
+    }
+  }
+}
+```
+
+## NOTES
+
+This rule is only applicable for Azure Database for MySQL servers with `'General Purpose'` and `'Memory Optimized'` tiers. The `'Basic'` tier does not support geo-redundant backup storage.
```

Comment:
Similar to Front Door there is a separate resource for MySQL Single server and MySQL Flexible (`Microsoft.DBforMySQL/flexibleServers`). Currently all the rules only support the single server deployment model.

So currently I think we should just note that this rule only applies to Single server.

```suggestion
This rule is only applicable for Azure Database for MySQL servers deployed in the Single Server deployment model with `'General Purpose'` and `'Memory Optimized'` tiers. The `'Basic'` tier does not support geo-redundant backup storage.

Currently this rule does not run against Azure Database for MySQL using the Flexible Server deployment model.
```

Reply:
@BernieWhite 

Great. Little bit off-topic in this PR, but do you think it makes sense to recommend customers using single server to migrate to flexible with some good documentation around it?

Given this statement:

![image](https://user-images.githubusercontent.com/99641908/200856031-a8c5d760-d71d-4c92-8050-4c7eeeb1fecb.png)